charts/tempo-distributed/values.yaml

```diff
@@ -825,6 +825,11 @@ config: |
       - {{ include "tempo.fullname" . }}-gossip-ring
   overrides:
     {{- toYaml .Values.global_overrides | nindent 2 }}
+    {{- if .Values.metricsGenerator.enabled }}
+    metrics_generator_processors:
+    - service-graphs
```

Comment:
eventually we will want this configurable, but this is fine for now.

Reply:
Agreed. I'm still learning my way around this stuff. so baby steps :)